File: Starters/Starters 48/FAIRPASS/Solution.c
#include <stdio.h>

int main() {
	int t,i,n,k;
	scanf("%d",&t);
	for(i=0;i<t;i++){
	    scanf("%d%d",&n,&k);
	    if(k>n) printf("YES\n");
	    else printf("NO\n");
	}
	return 0;
}

File: Starters/Starters 48/FAIRPASS/Solution.go
package main
import "fmt"

func main(){
  var t,n,k int
  fmt.Scanln(&t)
  for i:=0;i<t;i++ {
    fmt.Scan(&n,&k)
    if k>n {
      fmt.Println("YES")
    } else {
      fmt.Println("NO")
    }
  }
}
